### PR TITLE
describing prerequisities on Windows and macOS

### PR DESCRIPTION
**Pull request by semorrison**
I've attempted to describe known prerequisites, to smooth installation woes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,17 @@ curl https://raw.githubusercontent.com/Kha/elan/master/elan-init.sh -sSf | sh
...
 
+# Prerequisites
+
+On some systems `elan` will not work out of the box:
+
+* Windows 10
+   * You'll need a unix-like terminal.
+      * "git bash", available via [Git for Windows](https://gitforwindows.org/) works well, or
+      * the terminal from [MSYS2](https://www.msys2.org/), and then run `pacman -S unzip`
+   * [Visual C++ redistributable](https://www.microsoft.com/en-au/download/details.aspx?id=48145) (if you've already installed other software you likely have this).
```
Comment (Kha) [inline README.md]: @semorrison This should be fixed in elan 0.7.1. Do you still have any computer without the runtime left to give it a test? :)

File: README.md [hunk 2]
```diff
@@ -49,6 +49,17 @@ curl https://raw.githubusercontent.com/Kha/elan/master/elan-init.sh -sSf | sh
...
+* macOS: You'll need to install `libgmp`. Install [Homebrew](https://brew.sh/), then run `brew install libgmp`.
```
Comment (Kha) [inline README.md]: Perhaps add "to run Lean" after "libgmp" to clarify that this is not an issue with elan itself

File: README.md (hunk 1 above)
Comment (semorrison) [reply README.md]: I uninstalled the runtime, and tried again with 0.7.1. It seems to work.

I'm planning on making a fresh windows box sometime anyway, so I'll double check then.

File: README.md (hunk 2 above)
Comment (semorrison) [reply README.md]: I've clarified this.

**Comment by Kha** [comment]
Thanks!

**Pull request merged**